test/functional/device/otherApps-e2e-specs.js

```diff
@@ -1,37 +1,39 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { MOCHA_TIMEOUT, initSession, deleteSession } from '../helpers/session';
-import { MULTIPLE_APPS } from '../desired';
+import { MULTIPLE_APPS, amendCapabilities } from '../desired';
 import { translateDeviceName } from '../../../lib/utils';
 
 chai.should();
 chai.use(chaiAsPromised);
 
-describe('OtherApps', function () {
+describe('OtherApps', function() {
   this.timeout(MOCHA_TIMEOUT);
 
   let caps;
 
   let driver;
-  before(function () {
-    caps = Object.assign({
-      usePrebuiltWDA: true,
-      wdaStartupRetries: 0,
-    }, MULTIPLE_APPS);
-    caps.deviceName = translateDeviceName(caps.platformVersion, caps.deviceName);
+  before(function() {
+    caps = amendCapabilities(MULTIPLE_APPS, {
+      'appium:usePrebuiltWDA': true,
+      'appium:wdaStartupRetries': 0,
+      'appium:deviceName': translateDeviceName(
+        MULTIPLE_APPS.alwaysMatch['appuim:platformVersion'],
+        MULTIPLE_APPS.alwaysMatch['appium:deviceName']),
+    });
   });
 
-  afterEach(async function () {
+  afterEach(async function() {
     // try to get rid of the driver, so if a test fails the rest of the
     // tests aren't compromised
     await deleteSession();
   });
 
   if (!process.env.REAL_DEVICE) {
```

Comment:
this condition could be removed, it's not used anymore

Reply:
removed https://github.com/appium/appium-xcuitest-driver/pull/1461/commits/ea86bfe3cb1760e29fd1f92060269416bfe60253